src/deroff.rs

```diff
@@ -530,6 +533,27 @@ impl Deroffer {
     }
 }
 
+fn deroff_files(files: &[String]) -> std::io::Result<()> {
+    for arg in files {
+        eprintln!("{}", arg);
```

Comment:
Perhaps add some informational string before `{}` so it becomes easier to backtrack later from program output to location in source code?

Reply:
added!